programming/editors/vscode.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: page
+title: VSCode
+---
+
+# Visual Studio Code
+
+Visual Studio Code (known commonly as VSCode) is a free code editor developed by
```

Comment:
Feel like we should have a link to the VSCode home page on our first mention
```suggestion
[Visual Studio Code](https://code.visualstudio.com/) (known commonly as VSCode) is a free code editor developed by
```

Reply:
5889517